### ISSUE TITLE
ModelRunner failing assertion

### ISSUE THREAD

**Issue by jackm321** [pytorch/glow#1973]
example:
`$ ./bin/model-runner -model=shufflenet/model.onnx -dumpGraphDAG=shufflenetDAG.dot
Assertion failed: (tensors_.count(name) && "There is no tensor registered with this name."), function getTensorByName, file /Users/.../ProtobufLoader.cpp, line 33.
Abort trap: 6`

**Comment by jfix71** [comment]
I'm guessing this is because when we run shufflenet with the `image-classifier` it takes the `-model_input_name` on the command line which is used to create a placeholder for the external input (image) Tensor.

@jackm321 Why were you trying to run it with the `model-runner`? I think as is, the `model-runner` is only useful for models that have all Constant inputs and a single output.

**Comment by jackm321** [comment]
I was trying to generate one of the .dot graphs and this looked like the most simple binary that ran ModelLoader but I ended up just using `image-classifier` which worked like you mentioned. @jfix71 thanks for the explanation. So this isn't really an issue then I guess?

**Comment by jfix71** [comment]
@jackm321 Yeah, I think shufflenet only makes sense to run with `image-classifier` where you specify the Placeholder nodes. Otherwise you can't even build the graph to generate the dot file, as nodes are missing from the graph.
